### DISCUSSION TITLE
Can we put a real hard drive with Zulu on the SCSI?

### DISCUSSION THREAD

**Discussion by stephbb75** [ZuluSCSI/ZuluSCSI-firmware#306] [Q&A] [answered]
Hi,

I searched but couldn't find an answer, sorry if this has already been asked.
Can we put a real hard drive with Zulu on the SCSI?
It's a ZuluSCSI RP2040 with fireware 23.07.01-release
On an Amiga 4000T I put the zulu on my own, OK
Zulu and a CD player, ok
the Zulu and a hard drive, the hard drive is not seen (no ID or termination problem) and it does not read from any disk.

It works well otherwise, it's great for bringing an old machine back to life :-)

**Comment by PetteriAimonen** [comment, marked as answer]
Yes, as long as you assign different IDs to the drives and only the last drive on the bus has termination enabled, it should work.

One reason for problems could be that the TERMPWR line on the SCSI cable is running out of power. In that case, powering ZuluSCSI from the 4-pin connector or USB could help.

**Comment by stephbb75** [comment]
OK thanks,

My Zulu is powered by the 4 pin connector.
Could it be that the ZuluSCSI initializes faster than the hard drive and that this causes a problem?
